samples/nrf9160/gps/README.rst

```diff
@@ -0,0 +1,126 @@
+************
+
+* The following development board:
+
+  * |nRF9160DK|
+
+* Optional: SUPL Client library (for details on download, see :ref:`supl_client`)
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf9160/gps`
+
+.. include:: /includes/build_and_run_nrf9160.txt
+
+Using the sample with the SUPL Client library
+=============================================
+
+If the sample is to be used with SUPL client library, the library must be downloaded from the `Nordic Semiconductor website`_.
+See :ref:`supl_client` for information on installing and enabling the SUPL client library.
+
+Testing
+=======
+
+After programming the sample and all the prerequisites to the board, you can test the sample by performing the following steps:
+
+1. Connect your nRF9160 DK to the PC using a USB cable and power on or reset your nRF9160 DK.
+2. Open a terminal emulator.
+#. Test the sample by performing the following steps:
+
+    If SUPL client library support is not enabled:
+       a. Observe that the following information is displayed in the terminal emulator:
+
+          .. code-block:: console
+
+                Tracking: 0 Using: 0 Unhealthy: 0
+                Seconds since last fix 0
+
+                Scanning [/]
+
+       #. Observe that the numbers associated with the displayed parameters **Tracking** and **Using** change.
+       #. Observe that the sample displays the following on acquiring the first GPS lock:
+
+          .. code-block:: console
+
+		        Tracking: 7 Using: 7 Unhealthy: 0
+		        Seconds since last fix 0
+		        ---------------------------------
+		        Longitude:  10.437814
+		        Latitude:   63.421546
+		        Altitude:   163.747833
+		        Speed:      0.023171
+		        Heading:    0.000000
+		        Date:       30-01-2020
+		        Time (UTC): 09:42:38
+
+		        NMEA strings:
+		        $GPGGA,094238.25,6325.29275,N,01026.26884,E,1,08,1.49,163.75,M,0,,*2E
+		        $GPGLL,6325.29275,N,01026.26884,E,094238.25,A,A*66
+		        $GPGSA,A,3,05,07,08,09,16,21,27,30,,,,,2.60,1.49,2.12,1*17
+		        $GPGSV,2,1,8,7,60,093,44,9,14,106,39,21,17,339,41,16,07,016,22,1*5B
+		        $GPGSV,2,2,8,27,15,041,20,8,07,071,24,30,62,176,29,5,57,239,34,1*51
+		        $GPRMC,094238.25,A,6325.29275,N,01026.26884,E,0.05,0.00,300120,,,A,V*2E
+
+
+     If SUPL client library support is enabled:
+        a. Observe that the following message is displayed in the terminal emulator immediately after the device boots:
+
+           .. code-block:: console
+
+              New AGPS data requested, contacting SUPL server, flags 59.
+
+        #. Observe that the sample switches to LTE and connects to a SUPL server.
+        #. Observe that the A-GPS data gets downloaded.
+        #. Observe that the sample continues after the SUPL session is complete.
+
+Dependencies
+************
+
+This sample uses the following libraries:
+
+From |NCS|
+  * :ref:`secure_partition_manager`
```

Comment:
Plus:
at_cmd
at_notif
supl_os_client

Also, does it use any of the gps libraries?

Reply:
Ok, no gps library, only BSDLib for that